src/components/app/app.js

```diff
@@ -87,7 +87,12 @@ export default class App extends Component {
                   }}
```

Comment:
- Button for adding places doest work. places are not added to wish list.
- I think this size of buttons are too big for single page.
- Why did you decide to change view of buttons: in the very beginning there were 5 stars and now only 1, why did you write"With to visit", we used a looltip for this.
- Blue button wont fit for new design.

Reply:
- Three people checked - everything works well, I think the problem in your side, check your branches version and use git pull, also you need the last version of the server and log in to the application.
- In the dev branch, I don't see any changes in design, so I created these buttons as I like.
- because our average rating has floating-point value, and my decision has a sense
- it's not my work, I created a button for the previous version of App.

Reply:
- you saw changed on Andriy`s branch so you have to create smth that will fit for it.Because in any case it shoul be fixed after merging .
- What is not your work?That the  but button wasnt disabed

Reply:
I don't care what I saw. New design in dev - no, so stop spam here.
Your question about the blue color of the button, I answered that "it's not my work, I created a button for the previous version of App".
All your questions/comment have no sense just spam on GIT.

Reply:
Ok.

Reply:
It is not a SPAM, I ask questions according your work because it is my work. According rules of scrum and our process we shoul approve requests in the queue, Andriy request was opened earlier and it will be closed earlier so new design will be in dev before you.

Reply:
Ok ^_^